### DISCUSSION TITLE
Where are the section 7-8 YouTube videos for the ML Foundations course?

### DISCUSSION THREAD

**Discussion by ScoobyQ** [jonkrohn/ML-foundations#7] [Q&A] [answered]
Hi,
Does anyone have the links, please?

I can see the notebooks on GitHub:

- https://github.com/jonkrohn/ML-foundations/blob/master/notebooks/7-algos-and-data-structures.ipynb
- https://github.com/jonkrohn/ML-foundations/blob/master/notebooks/8-optimization.ipynb

The YT videos seem to end during section 6 with:
-  [Exercises on Event Probabilities — Topic 98 of Machine Learning](https://www.youtube.com/watch?v=GVTu_SK1X84)

**Comment by jonkrohn** [comment, marked as answer]
Hi @ScoobyQ, all of the details on where content is available at any given time is in the "Where and When" section of the repo's README.md (see [here](https://github.com/jonkrohn/ML-foundations#where-and-when)).

Essentially, I haven't had the bandwidth recently to record, produce and publish new home-recorded _ML Foundations_ content (which gets published to YouTube and Udemy). I hope to have the bandwidth to return to this soon and, yes, eventually all eight subjects will be available on YouTube and Udemy. To be sure not to miss when I start publishing on YouTube and Udemy again, you can sign up for my email newsletter on the homepage of [jonkrohn.com](https://www.jonkrohn.com/).

In the meantime, however, the publisher Pearson arranged for me to professionally record the entirety of my _ML Foundations_ content in a studio. This includes the subjects 7 and 8 that you're looking for. This is available via the O'Reilly learning platform [here](https://learning.oreilly.com/videos/the-essential-machine/9780137903245/). If you don't have access to O'Reilly (e.g., through your employer or educational institution), you can get a free 30-day trial with my special code `SDSPOD23`. Thirty days should be plenty of time to get through subjects 7 and 8 :)
